src/tests/auth/md5_password

```diff
@@ -1,7 +1,5 @@
 #
 #  over-ride password set in radiusd.conf
 #
-update control {
-	&Password.Cleartext -= ANY
-	&Password.With-Header := '{md5}5d41402abc4b2a76b9719d911017c592'
-}
+&control.Password.Cleartext -= ANY
```

Comment:
that doesn't work.  See my previous comments in other commits.  You must use

`&control -= &Password.Cleartext[*]`